regexes/device/mobiles.yml

```diff
@@ -17760,9 +17808,24 @@ Macoox:
 
 # Prixton
 Prixton:
-  regex: '(T7012Q)(?:[);/ ]|$)'
+  regex: 'PRIXTON |(T7012Q)(?:[);/ ]|$)'
   device: 'tablet'
```

Comment:
if `PRIXTON `  is more likely to match smartphones might be good to change the default

Reply:
the manufacturer now sells more tablets than smartphones

tablet useragents
```
"Mozilla/5.0 (Linux; Android 4.4.2; T1800Q+Prixton) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.143 Safari/537.36"
"Mozilla/5.0 (Linux; Android 4.4.2; T7014Q PRIXTON) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.132 Safari/537.36"
"Mozilla/5.0 (Linux; Android 5.1.1; T7014Q+PRIXTON) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.90 Safari/537.36"
"Mozilla/5.0 (Linux; Android 5.1.1; T1700Q PRIXTON Build/LMY47V; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/69.0.3497.100 Safari/537.36"
"Mozilla/5.0 (Linux; Android 5.1.1; T1700Q-PRIXTON) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.111 Safari/537.36"
```

Reply:
ok. it's fine then